src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -248,9 +250,29 @@ private void setupKeyboardShortcuts() {
                     break;
                 }
             }
+            if (SHOW_RELATED_ISSUE_OR_PR.match(event) && ui.getBrowserComponent().isCurrentUrlIssue()) {
+                if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
+                    showRelatedIssueOrPR();
+                } else { // only for default. can remove if default key for MARK_AS_READ_CHANGES
+                    markAsRead();
+                }
+            }
         });
     }
 
+    private void showRelatedIssueOrPR() {
+        TurboIssue issue = listView.getSelectedItem().get();
+        Optional<String> relatedIssueNumber = listView.getSelectedItem().get().isPullRequest()
+            ? GithubURLPageElements.extractIssueNumber(listView.getSelectedItem().get().getDescription())
+            : ui.getBrowserComponent().getPRNumberFromIssue();
+        if (relatedIssueNumber.isPresent()){
+            ui.triggerEvent(
+                    new IssueSelectedEvent(issue.getRepoId(), 
+                       Integer.parseInt(relatedIssueNumber.get()), panelIndex, issue.isPullRequest())
```

Comment:
The `parseInt` could be done in `extractIssueNumber` instead, then we can deal with an int here.